JOB.env

```diff
@@ -1,4 +1,4 @@
-DEFRA_VERSION=1.2.10
+DEFRA_VERSION=1.2.11
```

Comment:
I think this version number should be updated to be the default in the Dockerfile as well.

Although I'm wondering whether those defaults are even required? Is there possibly a better way of handling this rather than having to update this version number in two places?

Reply:
We had a conversation offline about ways of reducing the manual steps, which could be implemented in a future PR.

In the meantime I've updated the Dockerfile to reflect the correct default version, thanks.